taxonomy/models.py

```diff
@@ -350,6 +350,16 @@ class JobSkills(TimeStampedModel):
         )
     )
 
+    industry = models.ForeignKey(
+        'Industry',
+        blank=True,
+        null=True,
+        on_delete=models.deletion.CASCADE,
```

Comment:
I think `models.deletion.SET_NULL` would a better option here.

Reply:
I thought about it as well but NULL here means a special meaning, i.e data Job->Skill associating without Industry I most appropriate action would be to delete record it there is not Industry anymore